front/client/components/Article.vue

```diff
@@ -78,8 +81,11 @@ import AddItemDialog from 'client/components/AddItemDialog.vue'
 export default class ArticleItem extends Vue {
   isDialogOpen: boolean = false
 
-  async asyncData () {
-    await this.$store.dispatch('categoryItem/loadCategoryItem')
+  async asyncData() {
+    const rawUrl = this.$route.params.category
```

Comment:
You dont need to access router params, just pass it as props. https://router.vuejs.org/guide/essentials/passing-props.html